src/Controller/Cliente/CadastrarClienteController.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Controller\Cliente;
+
+use App\Entity\Cliente;
+use App\Repository\ClienteRepository;
+use App\Service\ValidarCpfService;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Http\Authentication\AuthenticationUtils;
+
+class CadastrarClienteController extends AbstractController
+{
+    private ClienteRepository $clienteRepository;
+    private ValidarCpfService $validarCpfService;
+
+    public function __construct(ClienteRepository $clienteRepository, ValidarCpfService $validarCpfService)
+    {
+        $this->clienteRepository = $clienteRepository;
+        $this->validarCpfService = $validarCpfService;
+    }
```

Comment:
dá para fazer assim: `property promotion`

```suggestion
    public function __construct(
        private ClienteRepository $clienteRepository,
        private ValidarCpfService $validarCpfService
    ) {
    }
```

Reply:
se tiver mais lugares nesse pr, ajusta também se quiser...